osquery/tables/applications/posix/docker_api.h

```diff
@@ -0,0 +1,12 @@
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/tables.h>
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+namespace tables {
```

Comment:
nit: mind adding newlines here also? Further, the comments for what these functions do would be better suited in the header, as opposed to the implementation. Mind moving them into here as opposed to the `.cpp`?

Reply:
Added